packages/react/.eslintrc

```diff
@@ -1,4 +1,7 @@
 {
   "extends": ["../../.eslintrc", "plugin:react/recommended", "plugin:react-hooks/recommended"],
-  "plugins": ["react", "react-hooks"]
+  "plugins": ["react", "react-hooks"],
+  "rules": {
+    "@typescript-eslint/consistent-type-imports": "error"
```

Comment:
can we separate this into its own PR and turn this on everywhere?

Reply:
Yea it should be seperate pr.   This pr in general mostly exists just to try to solicit early feedback before I go about refactoring anything.   I was going to open an issue but this pr was quick to whip up and I figured it would be better than an issue

Reply:
removed from this pr I can open a seperate pr later this week to add it.   I'll see if there are any other lint rules useful to add too while I'm at it.   I will only add a lint rule if it has an autofixer and is useful